text/0754-default-managers.md

```diff
@@ -0,0 +1,456 @@
+We'll want to insert a section early on about how helpers can be "local" to or defined on
+components and controllers. Then, once there have been examples of the local/private helper,
+the existing content can continue to talk about the "global helper" -- explicitly differentiating
+between the local/private helper, rather then retaining the general "Helper Function" title as
+"Helper Functions" are all of these, rather than just the globally defines ones.
+
+### Additional Pages
+
+This RFC adopts a default modifier manager, which means that folks can use modifiers without
+the assistance of a third-party addon, such as ember-modifier. This means that we'll benefit from
+adding a page to the guides on modifiers, their philosophy, when to use them, etc. While the
+overall API for modifiers in general has been "not perfect", there has been a fair amount of
+experimentation with modifiers since the 3.11 - 3.16 (pre-octane) era. Function modifiers are
+significantly simpler than class-based modifiers, and the complexity of the specifics of class-
+based modifiers is where most of the hesitance around adding modifiers by default has been.
+
+The default component manager may not need its own guides-documentation, because the prevelance
+of `@glimmer/component` is so widespread, documenting yet-another-way-to-define-a-component may
+cause more harm than good.
+
+Existing users of Ember should be made aware of these capabilities, once implemented, via
+the release blog post, with some examples -- but folks watching developments in the ecosystem
+will likely be aware of ember-could-get-used-to-this and ember-modifier, which both implement
+some parts of this RFC, so the migration path for users of those addons should be straight-forward.
+Those addons _may_ want to deprecate their similar functionality after the proposed behavior here
+lands -- though, this RFC suggests implementation such that developers may still use both
+addons without disruption.
+
+
+
+## Drawbacks
+
+
+For helpers and modifiers, there could be some awkwardness around the last argument
+passed to the function, as the type signature may not match how the call-site expects
+the signature to be. Projects like [Glint](https://github.com/typed-ember/glint#readme)
+would be essential for helping with clarity here.
+
+For components, people may begin to ask if they even need `@glimmer/component`, and that's
+something that the core team may want to address -- some folks may feel like there is churn
+in the component base-class, and don't know "what is correct", etc. The two approaches are
+similar enough where we could introduce a migration via lint rule or codemod.
+
+This proposal eliminates the possibility of using functions as components with the curly
+bracket syntax. However, using functions as components would still be possible with angle bracket
+invocation.
+
+The difference between `<Component @handler={{this.handler}} />` and `<Component @handler={{this.handler 1}} />`
```

Comment:
In strict mode, this is a non-issue because ambiguous invocations _must_ be invoked via `{{ (this.handler) }}`.
But, idk when strict mode is going to be default, or if default managers should be a reward for setting strict mode in your app (optional-feature?) idk